### ISSUE TITLE
Scroll of Inifuss has a black bar

### ISSUE THREAD

**Issue by nebulus-aversus** [bolrog/d2dx#165]
https://imgur.com/a/oJQSsSf

Not important of course, might be a quick fix though.

**Comment by oGkspAz** [comment]
Are you running it with other mods?
The reason I ask is I am not using D2DX atm but plain SGD2FreeRes instead and I have the same issue.
D2 1.14d
SGD2FreeRes v3.0.2.5b
BaseMod 1.13.9
PlugY v14.03
D2LootFilter v0.6-alpha

I was wondering if it might even be BaseMod causing it.

**Comment by updawg** [comment]
D2DX incorporates Freezes.

**Comment by nebulus-aversus** [comment]
Yeah, I was running BaseMod as well. But it happens even without BaseMod. No other mods either.

**Comment by oGkspAz** [comment]
Must be SGD2FreeRes then as it ships with D2DX as well. I'll check with creator.

**Comment by IAmTrial** [comment]
Cause of bug is now known. I will be working on a bugfix for this issue.
